cosmos/templates/index.html

```diff
@@ -137,4 +137,34 @@ <h5 class="card-title">{{ news.title }}</h5>
         </div>
     </div>
 </section>
+<section id="partners" class="pb-5">
+    <div class="container py-5">
+        <div class="section-title">
+            <h2>Partners</h2>
+        </div>
+        <div class="row mx-auto my-auto justify-content-center partners">
+                <div id="partnerCarousel" class="carousel slide" data-bs-ride="carousel">
+                    <div class="carousel-inner" role="listbox">
+                        {% for partner in partners %}
+                        <div class="carousel-item{% if forloop.first %} active{% endif %}">
+                            <div class="col-md-4 text-center">
+                                <a href="{{ partner.url }}">
+                                    <img src="{{ partner.image.url }}" alt="{{ partner.name }}">
+                                </a>
+                            </div>
+                        </div>
+                        {% endfor %}
+                    </div>
+                    <button class="carousel-control-prev btn-cosmos" type="button" data-bs-target="#partnerCarousel">
+                        <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+                        <span class="visually-hidden">Previous</span>
+                    </button>
+                    <button class="carousel-control-next btn-cosmos" type="button" data-bs-target="#partnerCarousel">
+                        <span class="carousel-control-next-icon" aria-hidden="true"></span>
+                        <span class="visually-hidden">Next</span>
+                    </button>
+                </div>
+        </div>
+    </div>
+</section>
```

Comment:
Maybe make it so that this section is only shown when there exists a partner, cause cosmos often doesnt have a partner afaik.

Reply:
And also maybe make it a carousel only when it is needed, as in there are more than 3 partners, cause we can best avoid the hacky css to make it a carousel when possible.